### PR TITLE
feat(external videos): back end improvements

### PR DESCRIPTION
**Pull request by pedrobmarin**
Cherry picked some improvements we've been doing to the external videos feature.

- Give external videos it's own MongoDB collection. Similar on how is made for recording props.
- Add rights management at akka-apps and removes it from Meteor.
- Move to akka-apps the control on when to stop and close the feature: presenter change/leave and screenshare start.

Kudos to @frankemax and @Arthurk12

### CODE REVIEW

File: bigbluebutton-html5/imports/api/external-videos/server/handlers/startExternalVideo.js [hunk 1]
```diff
@@ -1,23 +1,13 @@
...
 export default function handleStartExternalVideo({ header, body }, meetingId) {
-  const { userId } = header;
+  check(header, Object);
   check(body, Object);
   check(meetingId, String);
-  check(userId, String);
 
-  const externalVideoUrl = body.externalVideoUrl;
-  const user = Users.findOne({ meetingId: meetingId, userId: userId })
+  const { userId } = header;
```
Comment (antobinary) [inline bigbluebutton-html5/imports/api/external-videos/server/handlers/startExternalVideo.js:9]: We don't check if `userId` and `externalVideoUrl` are strings neither here nor in akka-apps, we just start using them

File: bigbluebutton-html5/imports/api/external-videos/server/methods/startWatchingExternalVideo.js [hunk 2]
```diff
@@ -1,32 +1,26 @@
...
-  const { meetingId, requesterUserId: userId } = extractCredentials(this.userId);
-  const { externalVideoUrl } = options;
-
   try {
+    const { meetingId, requesterUserId } = extractCredentials(this.userId);
+
     check(meetingId, String);
-    check(userId, String);
+    check(requesterUserId, String);
     check(externalVideoUrl, String);
```
Comment (antobinary) [inline bigbluebutton-html5/imports/api/external-videos/server/methods/startWatchingExternalVideo.js:16]: We check here but not on `startExternalVideo`. Is this sufficient?

**Review by antobinary — APPROVED** [review]
Tested locally and it worked well.
The check for string on share vs on consuming the url looks odd but it's still checked. We may tweak things in future.

File: bigbluebutton-html5/imports/api/external-videos/server/methods/startWatchingExternalVideo.js (hunk 2 above)
Comment (pedrobmarin) [reply bigbluebutton-html5/imports/api/external-videos/server/methods/startWatchingExternalVideo.js:16]: We pushed those checks to the modifier but the `userId` is missing, indeed.

**Pull request merged**